### PR TITLE
oah shell for KOB

### PR DESCRIPTION
**Pull request by asa1997**
oah shell with hyperledgerkochi namespace.

### CODE REVIEW

**Comment by asa1997** [comment]
@panickervinod Please do take a look at this.

File: src/bash/install.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-OAH_NAMESPACE=${OAH_NAMESPACE:=openapphack}
+OAH_NAMESPACE=${OAH_NAMESPACE:=hyperledgerkochi}
```
Comment (panickervinod) [inline src/bash/install.sh:3]: @asa1997 This cannot be merged back with openapphack namespace. 

Please avoid hardcoding the namespace to hyperledgerkochi, should be driven through export variable settings

File: src/bash/install.sh [hunk 2]
```diff
@@ -55,7 +55,7 @@ case "$(uname)" in
 esac
 
 echo '                                                                     '
-echo 'Thanks for using     OpenAppHack(OAH SHELL)                          '
+echo 'Thanks for using     Hyperledgerkochi(OAH SHELL)                            '
```
Comment (panickervinod) [inline src/bash/install.sh:58]: Hardcording here will make it difficult to merge back to openapphack

File: src/bash/install.sh [hunk 3]
```diff
@@ -307,7 +337,7 @@ EOF
 # echo "oah_insecure_ssl=false" >> "${oah_config_file}"
 
 echo "Download script archive..."
-#https://github.com/openapphack/oah/raw/gh-pages/
+#https://github.com/hyperledgerkochi/oah/raw/gh-pages/
```
Comment (panickervinod) [inline src/bash/install.sh:340]: @asa1997  remove hardcoding in the path.

File: src/bash/oah-init.sh [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-export OAH_NAMESPACE=${OAH_NAMESPACE:=openapphack}
+export OAH_NAMESPACE=${OAH_NAMESPACE:=hyperledgerkochi}
```
Comment (panickervinod) [inline src/bash/oah-init.sh:3]: @asa1997 avoid overriding the default value in the code, use export variable to drive custom name space.

File: src/bash/oah-install.sh [hunk 5]
```diff
@@ -91,8 +91,8 @@ function __oah_install_candidate_version {
 	# version defaults to master if no tag is giving
 	VERSION="${2:=master}"
 
-	OAH_GIT_URL="http://github.com/openapphack/"
-	CANDIDATE_GIT_REPO_URL="${3:="http://github.com/openapphack/"}"
+	OAH_GIT_URL="http://github.com/hyperledgerkochi/"
```
Comment (panickervinod) [inline src/bash/oah-install.sh:94]: @asa1997 avoid hardcoding of namespace

File: src/bash/selfupdate.sh [hunk 6]
```diff
@@ -33,7 +33,7 @@
 # # 	OAH_DIR="$HOME/.oah"
 # # fi
 #
-# OAH_NAMESPACE=${OAH_NAMESPACE:=openapphack}
+# OAH_NAMESPACE=${OAH_NAMESPACE:=hyperledgerkochi}
```
Comment (panickervinod) [inline src/bash/selfupdate.sh:36]: @asa1997 avoid overriding the default value in the code, use export variable to drive custom name space.

**Review by panickervinod — COMMENTED** [review]
@asa1997 avoid overriding the default value of namespace in the code, use custom installer script with export variable to drive custom name space.

**Comment by panickervinod** [comment]
asa1997 avoid overriding the default value in the code, use export variable and custom install script to drive custom name space based installation.

**Comment by asa1997** [comment]
@panickervinod Ok Vinod

**Pull request closed without merging**